episodes/introduction.md

```diff
@@ -89,10 +179,24 @@ All of the above are things a Hub Champion should do to enable their community t
 
 ::::
 
+## Next Steps
+
+The next part of the training is a self-guided study of the following episodes
+
+-. {ref}`episode-admin`
+-. {ref}`episode-images`
+-. {ref}`episode-data`.
+
+and concludes with an in-person episode covering {ref}`episode-troubleshoot`.
+
+To complete the self-study portion of the workshop, check that you can access the [2i2c Community Showcase Hub](https://showcase.2i2c.cloud) and write your answers to the exercises in our [collaborative note-taking document](#).
```

Comment:
```suggestion
To complete the self-study portion of the workshop, you will need to be able to access the [2i2c Community Showcase Hub](https://showcase.2i2c.cloud). When working through the self-study sections of the workshop, write your answers to the exercises in our [collaborative note-taking document](#).
```

Reply:
You may also want to add a sentence describing what learners should do if they find that they cannot access that Showcase Hub.